app/modules/collaborations/models.py

```diff
@@ -17,13 +51,161 @@ class Collaboration(db.Model, HoustonModel):
     guid = db.Column(
         db.GUID, default=uuid.uuid4, primary_key=True
     )  # pylint: disable=invalid-name
-    title = db.Column(db.String(length=50), nullable=False)
+    title = db.Column(db.String(length=50), nullable=True)
+
+    collaboration_user_associations = db.relationship(
+        'CollaborationUserAssociations', back_populates='collaboration'
+    )
+
+    def __init__(
+        self, user_guids=None, approval_states=None, is_initiator=None, **kwargs
+    ):
+        # I think this can be adapted to allow >2 User collabs, opening the door for Org-level.
+        if user_guids is None or len(user_guids) != 2:
+            raise ValueError(
+                '__init__ Collaboration: Creating a collaboration requires a user_guids list.'
+            )
+
+        # You can create a collaboration where both users 'approved'
+        if approval_states is not None and len(approval_states) != len(user_guids):
+            raise ValueError(
+                '__init__ Collaboration: Length of approval_states was not equal to provided user list.'
+            )
+
+        if is_initiator is not None and len(is_initiator) != len(user_guids):
```

Comment:
pedantic but should is_initiator actually be are_initiators as it seems inherantly plural?

Reply:
Yep, that's unclear. Changing to initator_states to be similar to approval_states.